Modules/FIT/FT0/src/TH1ReductorLaser.cxx

```diff
@@ -34,18 +34,25 @@ void TH1ReductorLaser::update(TObject* obj)
   if (auto histo = dynamic_cast<TH2F*>(obj)) {
     int channel = -1;
     sscanf(histo->GetName(), "%*[^0-9]%d", &channel);
+    std::cerr << histo << std::endl;
```

Comment:
```suggestion
```
Please avoid using directly stderr, this will be anyway redirected to infologger, but as warnings.